arviz/__init__.py

```diff
@@ -1,8 +1,20 @@
 # pylint: disable=wildcard-import,invalid-name,wrong-import-position
 """ArviZ is a library for exploratory analysis of Bayesian models."""
 __version__ = "0.2.1"
+from matplotlib import get_backend as mpl_get_backend
```

Comment:
I dont think we should include this import for two reasons. One is that matplotlib should handle its own logging, the other is that this adds another object to the namespace